NEWS.md

```diff
@@ -1,3 +1,7 @@
+## dde 1.0.1
+
+* Several memory errors fixed [#22](https://github.com/mrc-ide/dde/pull/22)
+
```

Comment:
Perhaps a link to the reside blog about this.

Reply:
I'll retrospectively add it to avoid issues with the CRAN link checker